tests/test_api_trusted.py

```diff
@@ -1096,3 +1103,65 @@ def test_remapping_after(self):
             self.assertTrue(str(team) in {'5'})
         for team in Award.get_by_id('2014casj_2').recipient_dict.keys():
             self.assertTrue(str(team) in {'6'})
+
+    def test_zebra_motionworks_add(self):
```

Comment:
Should we add test cases for non-success states as well?

Reply:
Yep, I'll add some.